tuplex/core/include/logical/FileInputOperator.h

```diff
@@ -56,15 +56,33 @@ namespace tuplex {
 
         void detectFiles(const std::string& pattern);
 
+        // CSV Constructor
```

Comment:
let's maybe add a TODO here, that we really should refactor this. These constructors are a mess.